types/qunit/index.d.ts

```diff
@@ -426,7 +426,7 @@ declare global {
          *
          * @callback callback Callback to execute.
          */
-        begin(callback: (details: QUnit.BeginDetails) => void): void;
+        begin(callback: (details: QUnit.BeginDetails) => void): void | Promise<void>;
```

Comment:
I think you want:
```suggestion
        begin(callback: (details: QUnit.BeginDetails) => void | Promise<void>): void;
```
(for all of these)

Reply:
Oops, you are correct!